src/services/syntaxServices.js

```diff
@@ -0,0 +1,19 @@
+services.factory('syntaxServices', [function() {
```

Comment:
I'd change the name to nifSyntaxServices or something like that so it can be guessed in a moment the syntax of what you are validating.